DMX.Gatekeeper.Api.Tests.Acceptance/APIs/Homes/HomeApiTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DMX.Gatekeeper.Api.Tests.Acceptance.Brokers;
+using FluentAssertions;
+using Xunit;
+
+namespace DMX.Gatekeeper.Api.Tests.Acceptance.APIs.Homes
+{
+    [Collection(nameof(ApiTestCollection))]
+    public class HomeApiTests
+    {
+        private readonly DmxGatekeeperApiBroker dmxGatekeeperApiBroker;
+
+        public HomeApiTests(DmxGatekeeperApiBroker dmxGatekeeperApiBroker)
+        {
+            this.dmxGatekeeperApiBroker = dmxGatekeeperApiBroker;
```

Comment:
This can be fat-arrowed.

here's a rule:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Methods.md#110-one-liners